src/components/o-modal/index.js

```diff
@@ -48,6 +48,9 @@ class AXAModal extends BaseComponentGlobal {
   connectedCallback() {
     super.connectedCallback();
 
+    // FOUC workaround
+    this.removeAttribute('style');
```

Comment:
and here classList.remove()

Reply:
This only works if said modifier is externel to `<axa-modal>`'s CSS and loaded within the `<head>` before anything else 💣

Reply:
better to remove just the specific `display` style property by using these APIs:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Properties_Reference

Reply:
agree with andy

Reply:
@raphaellueckl please can u do as @AndyOGo said here (`wcNode.style.display = 'block';`)?

Reply:
@LucaMele @AndyOGo Done, please have a look again.